domain/schema/model/sql/0024-relation.sql

```diff
@@ -84,20 +84,20 @@ ON relation (relation_id);
 -- The relation_unit table links a relation to a specific unit.
 CREATE TABLE relation_unit (
     uuid TEXT NOT NULL PRIMARY KEY,
-    relation_uuid TEXT NOT NULL,
+    relation_endpoint_uuid TEXT NOT NULL,
     unit_uuid TEXT NOT NULL,
     in_scope BOOLEAN DEFAULT FALSE,
```

Comment:
We need to remove this `in_scope` field. A unit is in the relation scope by virtue of having a record in this table. 

So there is also no `upsertRelationUnitAndEnterScope` (above) either. It's one thing - inserting the relation unit.

Someone has gotten confused reading the old state code. When you call `Relation.Unit` there, it does not imply that here is any scope record for that unit in that relation, it creates a data structure, which as one of its functions can check if the unit is `InScope` by looking for it in the `relationscopes` collection.